stock_quant_manual_assign/wizard/assign_manual_quants.py

```diff
@@ -43,17 +43,25 @@ def _compute_qties(self):
 
     @api.multi
     def assign_quants(self):
+        quant = self.env['stock.quant']
         move = self.env['stock.move'].browse(self.env.context['active_id'])
-        move.picking_id.mapped('pack_operation_ids').unlink()
-        quants = []
-        # Mark as recompute pack needed
-        if move.picking_id:  # pragma: no cover
-            move.picking_id.recompute_pack_op = True
-        for quant_id in move.reserved_quant_ids.ids:
-            move.write({'reserved_quant_ids': [[3, quant_id]]})
-        for line in self.quants_lines.filtered('selected'):
-            quants.append([line.quant, line.qty])
-        self.env['stock.quant'].quants_reserve(quants, move)
+        move._do_unreserve()
+        for line in self.quants_lines:
+            if line.qty > 0:
```

Comment:
@fanha99 Please use float_compare

Reply:
should I use math.isclose(line.qty, 0.0, rel_tol=1e-6)
the math.isclose is with the python 3.5